packages/openapi-fetch/package.json

```diff
@@ -47,19 +40,18 @@
     "svelte"
   ],
   "scripts": {
-    "build": "pnpm run build:clean && pnpm run build:js && pnpm run build:js-min && pnpm run build:cjs",
-    "build:clean": "del-cli dist",
-    "build:js": "mkdir -p dist && cp src/* dist",
-    "build:js-min": "esbuild --bundle src/index.js --format=esm --minify --outfile=dist/index.min.js && cp dist/index.d.ts dist/index.min.d.ts",
-    "build:cjs": "esbuild --bundle src/index.js --format=cjs --outfile=dist/cjs/index.cjs && cp dist/index.d.ts dist/cjs/index.d.cts",
+    "build": "unbuild",
     "format": "biome format . --write",
-    "lint": "biome check .",
+    "lint": "pnpm run lint:js && pnpm run lint:ts && pnpm run lint:ts-no-strict",
+    "lint:js": "biome check .",
+    "lint:ts": "tsc --noEmit",
+    "lint:ts-no-strict": "tsc --noEmit -p test/no-strict-null-checks/tsconfig.json",
     "generate-types": "openapi-typescript -c test/redocly.yaml",
+    "prepack": "pnpm run build",
     "pretest": "pnpm run generate-types",
-    "test": "pnpm run \"/^test:/\"",
+    "test": "pnpm run test:js && pnpm run test:exports",
     "test:js": "vitest run",
-    "test:ts": "tsc --noEmit",
-    "test:ts-no-strict": "tsc --noEmit -p test/no-strict-null-checks/tsconfig.json",
+    "test:exports": "pnpm run build && attw --pack .",
```

Comment:
I really like the “are the types wrong” type tests, so added to all packages. We were also able to remove the `cjs-resolves-to-esm` rule 😎